Makefile

```diff
@@ -37,9 +37,9 @@ endif
 
 APP_DEPLOYER_IMAGE ?= $(REGISTRY_PREFIX)/deployer:$(TAG)
 CONJUR_IMAGE ?= $(REGISTRY)/$(PREFIX):$(TAG)
-POSTGRES_SOURCE_IMAGE ?= postgres:10.1
+POSTGRES_SOURCE_IMAGE ?= postgres:12.2
```

Comment:
Given that we don't have good testing around this suite, I think we need to leave PG as-is or even dial back to 9.4 that we use everywhere unless you want to manually test out authn-k8s in the chart. `container-dap` repo has the most secure images but they may need manual assembly and that may not be desired here.

Reply:
Yes, totally agree after seeing @izgeri and your comments.
We should dial back to postgres:9.4 to be consistent with cyberark/conjur, and postgres:9.4 has less critical vulnerabilites.

Reply:
Sounds good to me!